docs/content/wiring.md

```diff
@@ -153,6 +153,32 @@ For example, given the install step above, we can use the `MYSQL_URL` with the h
 
 Just like in the case of credentials and parameters, the value of the `bundle.outputs.MYSQL_URL` reference will be rewritten in the YAML before the helm mixin is invoked.
 
+## Wiring Images
+
+In the `porter.yaml`, you can define what images will be used within the bundle with the `images` section:
+
+```yaml
+images:
+  ALIAS:
+    description: A very useful image
+    imageType: docker # porter.yaml can default this to docker if we aren't already
+    repository: gcr.io/mcguffin-co/mcguffin
+    digest: sha256:85b1a9 # this is what bundle.json allows
+    tag: v1.1.0 # we can collect this and make it available but it won't land into bundle.json
```

Comment:
It won't land into bundle.json b/c the digest sha is provided?  Also, is the digest key/value mandatory or optional?  (Or more generally, is it worth explicitly mentioning which fields are mandatory?)

Reply:
I think this is probably better in a separate doc about images? What say you @carolynvs. We added that but never really documented it (there was a comment that we did nothing with it before). I was going to open a more general "document how to use `images`" issue that we could give best practice info in and explain how to use for Helm charts, etc.